src/ray/gcs/gcs_client/test/gcs_client_test.cc

```diff
@@ -878,6 +903,10 @@ TEST_P(GcsClientTest, TestWorkerTableResubscribe) {
 }
 
 TEST_P(GcsClientTest, TestGcsTableReload) {
+  // Restart gcs only work with redis.
+  if (RayConfig::instance().bootstrap_with_gcs()) {
```

Comment:
What will it take to set up external Redis for this test, in a future PR?

Reply:
I think in the end, we need to test with two groups of setups for different storage.